src/classes/CRLP_Query_SEL.cls

```diff
@@ -64,8 +64,13 @@
             allFieldsToQuery.add(SObjectType.Opportunity.fields.Amount.getSobjectField().getDescribe());
             allFieldsToQuery.add(SObjectType.Opportunity.fields.CloseDate.getSobjectField().getDescribe());
             allFieldsToQuery.add(SObjectType.Opportunity.fields.npe03__Recurring_Donation__c.getSobjectField().getDescribe());
-        } else  if (resultObject == npe01__OppPayment__c.SObjectType) {
+        } else if (resultObject == npe01__OppPayment__c.SObjectType) {
             allFieldsToQuery.add(SObjectType.npe01__OppPayment__c.fields.npe01__Opportunity__c.getSobjectField().getDescribe());
+        } else if (resultObject == npe03__Recurring_Donation__c.SObjectType) {
+            allFieldsToQuery.add(SObjectType.npe03__Recurring_Donation__c.fields.npe03__Next_Payment_Date__c.getSobjectField().getDescribe());
```

Comment:
There shouldn't be a need to hardcode the target recurring donation fields here. This should just pick up fields that are not referenced by the CMT records for the given rollup type.